packages/server-wallet/src/wallet/store.ts

```diff
@@ -334,6 +335,8 @@ export class Store {
       return {
         channelIds: stateChannelIds.concat(objectiveChannelIds),
         channelResults,
+        // HACK: This will possibly cause the wallet to re-emit `'objectiveStarted'` multiple times
```

Comment:
This probably should be an issue? (If not one already)

Reply:
I was waiting until this PR got accepted.

Reply:
It's not obvious to me why this will cause the event to be emitted multiple times? Can you explain?

Reply:
> It's not obvious to me why this will cause the event to be emitted multiple times? Can you explain?

48443417b610cf2a792510a91b3c78d450daf712

> This probably should be an issue? (If not one already)

https://github.com/statechannels/statechannels/issues/3289